controllers/podchaos/types.go

```diff
@@ -40,7 +46,37 @@ func (r *Reconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
 		r.Log.Error(err, "unable to get podchaos")
 		return ctrl.Result{}, nil
 	}
+	scheduler := podchaos.GetScheduler()
+	duration, err := podchaos.GetDuration()
+	if err != nil {
+		r.Log.Error(err, "unable to get duration")
+		return ctrl.Result{}, nil
+	}
+	if scheduler == nil && duration == nil {
+		return r.durationPodChaos(&podchaos, req)
```

Comment:
maybe you should give it a better name than `durationPodChaos`?

Reply:
updated.